charts/dvpe-deployment-gloo/values.yaml

```diff
@@ -363,6 +363,12 @@ istio:
 certificate:
       # certificate.useCustomIssuer -- If set to `true` the default issuer's configuration settings will be used. This requires a custom issuer to be installed.
       useCustomIssuer: true
+      # certificate.customIssuerSel -- custom issuer selector
```

Comment:
I would rather replace this with a single flag like for example

issuerType: <intranet/internet>

The kind will always be ClusterIssuer and can be hard coded since we won't provide any other sort of issuers in the future.